authors/apps/articles/tests/base_test.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework.test import APITestCase, APIClient
+from rest_framework.views import status
+from django.urls import reverse
+
+
+class BaseTest(APITestCase):
```

Comment:
You can reuse the `BaseTest` in `authentication` by inheriting it and adding any methods specific to testing articles.